content/contest.md

```diff
@@ -7,24 +7,23 @@ menu: main
 type: page
 ---
 # The Contest
-The Northwestern Europe Regional Contest (NWERC) is a contest in which teams from universities all over the Northwestern part of Europe are served a series of algorithmic problems. The goal of each team is to solve as many problems as possible within the 5 hour time limit. Potential solutions are submitted and corrected by an automated judging system. The team that solves the most problems at the end of the contest qualify for the ICPC World Finals.
+The Northwestern Europe Regional Contest (NWERC) is a contest in which teams from universities all over the Northwestern part of Europe are served a series of algorithmic problems. The goal of each team is to solve as many problems as possible within the 5 hour time limit. Potential solutions are submitted and corrected by an automated judging system. The teams that solve the most problems at the end of the contest qualify for the ICPC World Finals.
```

Comment:
officially it is team(non-plural)

Reply:
Ah, then I fixed it the wrong way around :stuck_out_tongue: Thanks! :smile: